README.md

```diff
@@ -1,8 +1,11 @@
 # referer-parser Java/Scala library
 
+[![Build Status](https://travis-ci.org/snowplow-referer-parser/jvm-referer-parser.svg?branch=develop)](https://travis-ci.org/snowplow-referer-parser/jvm-referer-parser)
+[![codecov](https://codecov.io/gh/snowplow-referer-parser/jvm-referer-parser/branch/master/graph/badge.svg)](https://codecov.io/gh/snowplow-referer-parser/jvm-referer-parser)
```

Comment:
can we add placeholder badges for maven central and gitter, I'm working on creating the gitter room